src/txn/txn_ext.c

```diff
@@ -101,5 +101,5 @@ __wt_ext_transaction_visible(
 	(void)wt_api;					/* Unused parameters */
 
 	return (__wt_txn_visible(
-	    (WT_SESSION_IMPL *)wt_session, transaction_id));
+	    (WT_SESSION_IMPL *)wt_session, transaction_id, NULL));
```

Comment:
Yes, I've added a section about the extension API after the main part of the timestamp documentation.